server/src/main/scala/org/ensime/server/Server.scala

```diff
@@ -129,7 +129,7 @@ object Server extends AkkaBackCompat {
       throw new RuntimeException(s".ensime file ($ensimeFile) not found")
 
     implicit val config: EnsimeConfig = try {
-      EnsimeConfigProtocol.parse(Files.toString(ensimeFile, Charsets.UTF_8))
+      EnsimeConfigProtocol.parse(new String(Files.readAllBytes(ensimeFile.toPath), StandardCharsets.UTF_8))
```

Comment:
I'm sure there is a better way to do this by adding the functionality onto EnsimeRawFile